### PR TITLE
More link using the manager + dayview link

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#133011](http://central.tri.be/issues/133011)
---
http://i.bordoni.me/ff90d8113a9f

### CODE REVIEW

**Comment by paulmskim** [comment]
@bordoni tests are failing

**Comment by juanfra** [comment]
@bordoni I'm adding the day link via template vars on this one: https://github.com/moderntribe/the-events-calendar/pull/2665 - Maybe we can use the same for this link?

**Comment by bordoni** [comment]
@juanfra tests are passing.

File: tests/views_integration/Tribe/Events/Views/V2/HTML/MonthTest.php [hunk 1]
```diff
@@ -75,6 +75,8 @@ protected function given_month_data() {
 				'multiday_events'  => [],
 				'found_events'     => 0,
 				'more_events'      => 0,
+				'more_events'      => 0,
```
Comment (Camwyn) [inline tests/views_integration/Tribe/Events/Views/V2/HTML/MonthTest.php]: duplicate?

Comment (bordoni) [inline tests/views_integration/Tribe/Events/Views/V2/HTML/MonthTest.php]:
```suggestion
```

**Pull request merged**